Playbooks/Enrich-AzureResourceGraph/azuredeploy.json

```diff
@@ -0,0 +1,630 @@
+                            "runAfter": {
+                                "Condition": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "ApiConnection",
+                            "inputs": {
+                                "host": {
+                                    "connection": {
+                                        "name": "@parameters('$connections')['keyvault']['connectionId']"
+                                    }
+                                },
+                                "method": "get",
+                                "path": "/secrets/@{encodeURIComponent('azureresourcegraph-clientid')}/value"
+                            }
+                        },
+                        "Get_secret_azureresourcegraph-clientsecret": {
+                            "runAfter": {
+                                "Get_secret_azureresourcegraph-clientid": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "ApiConnection",
+                            "inputs": {
+                                "host": {
+                                    "connection": {
+                                        "name": "@parameters('$connections')['keyvault']['connectionId']"
+                                    }
+                                },
+                                "method": "get",
+                                "path": "/secrets/@{encodeURIComponent('azureresourcegraph-clientsecret')}/value"
+                            },
+                            "runtimeConfiguration": {
+                                "secureData": {
+                                    "properties": [
+                                        "outputs"
+                                    ]
+                                }
+                            }
+                        },
+                        "HTTP_Azure_Login": {
+                            "runAfter": {
+                                "Get_secret_azureresourcegraph-clientsecret": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "Http",
+                            "inputs": {
+                                "body": "resource=https://management.azure.com&grant_type=client_credentials&client_id=@{body('Get_secret_azureresourcegraph-clientid')?['value']}&client_secret=@{body('Get_secret_azureresourcegraph-clientsecret')?['value']}",
+                                "headers": {
+                                    "Content-Type": "application/x-www-form-urlencoded"
+                                },
+                                "method": "POST",
+                                "uri": "https://login.microsoftonline.com/29208c38-8fc5-4a03-89e2-9b6e8e4b388b/oauth2/token"
```

Comment:
Tenant "29208c38-8fc5-4a03-89e2-9b6e8e4b388b" is hard coded. Will it work for other tenants?

Reply:
fixed with Playbooks/Notify-ASCAlertAzureResource/azuredeploy.json as model